client/my-sites/plans-features-main/hooks/use-experiment-for-trail-map.ts

```diff
@@ -1,45 +1,47 @@
 import config from '@automattic/calypso-config';
+import {
+	isTrailMapAnyVariant,
+	isTrailMapCopyVariant,
+	isTrailMapStructureVariant,
+	setTrailMapExperiment,
+	TrailMapVariantType as VariantType,
+} from '@automattic/calypso-products';
+import { useEffect } from 'react';
 import { useExperiment } from 'calypso/lib/explat';
-import type { DataResponse } from '@automattic/plans-grid-next';
 
-interface Props {
+interface Params {
 	flowName?: string | null;
 }
 
-export const TrailMapVariant = {
-	Control: 'control',
-	TreatmentCopy: 'treatment-copy',
-	TreatmentStructure: 'treatment-structure',
-	TreatmentCopyAndStructure: 'treatment-copy-and-structure',
-} as const;
```

Comment:
I think we can achieve the same type safety by just using the variant strings and avoiding the need for this object.